extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientBuildTimeConfig.java

```diff
@@ -17,4 +17,10 @@
      */
     @ConfigItem(name = "metrics.enabled")
     public boolean metricsEnabled;
+
+    /**
+     * If set to true, the default clients will always be created even if there are no injection points that use them
+     */
+    @ConfigItem(name = "default-clients.force")
```

Comment:
A wise man said to me, don't use dots in config item names.

Also I don't see why `force-default-clients` would be a bad name.

Reply:
Basically, the reason for no dots is that it doesn't work. I got hit by this in the Hibernate ORM config. No errors but it fails if you try to define the value in the application.properties.

If you really want a dot, use a group.

Reply:
Understood, I'll change it